core/src/main/java/dk/alexandra/fresco/framework/builder/BuilderFactoryNumeric.java

```diff
@@ -37,4 +40,8 @@ default AdvancedNumericBuilder createAdvancedNumeric(ProtocolBuilderNumeric buil
   default SequentialNumericBuilder createProtocolBuilder() {
     return ProtocolBuilderNumeric.createApplicationRoot(this);
   }
+
+  default UtilityBuilder createUtilityBuilder(ProtocolBuilderNumeric builder) {
```

Comment:
UtilityBuilder is not necessarily the best wording - utlity is very generic, and this interface has a very specific purpose. Maybe DebugBuilder or DebugViaOpenBuilder

Actually it is not needed to make as a builder, it consist of four different ComputationBuilders that can be inlined whereever used. No need (nor reason) to make it a part of the BuilderFactoryNumeric interface

Reply:
While this is true in theory, I suspect new users will never ever find these helpful debug methods. That is the main reason why I included it. 
I will, however, rename it to DebugBuilder and include javadoc to warn against using it in production.

Reply:
I would still very much insist it is not available at all in production code.
How about leaving the class as is, i.e. leave the the artist formerly known as DefaultUtilityBuilder, but remove the interface from the BuilderFactoryNumeric. Then there is a perfectly fine debugger - but now it is not hardwired into the interface - but instantiable everywhere,.